### ISSUE TITLE
AAA team stream URL via 'team' argument returning sample stream/BBB video

### ISSUE THREAD

**Issue by cr08** [tonywagner/mlbserver#43]
Just now getting around to posting this but it seems like it began at the start of the season.

But if I call the stream URL with the team and level arguments, I get the demo stream/BBB video.

`http://192.168.1.3:9898/stream.m3u8?team=COL&level=AAA&resolution=best`

But if I manually pick the team ID, it plays fine.

Side question but will likely come into play for me once this is sorted: With the game 2 arguments, do those come into play for AAA games as well? Looking at the example links in theory for this team it should be `http://192.168.1.3:9898/stream.m3u8?team=COL.2&level=AAA&resolution=best` ?

Thanks for all the help and your work on this. This has been an amazingly useful tool especially for these minor league teams that don't normally air outside of MLB.TV

**Comment by tonywagner** [comment]
That first link plays fine for me today. Are you on the latest version?

Adding the ".2" to the team is only for national TV games. For the second game of a doubleheader, you want to add "&game=2" to the stream URL. It wasn't working for MiLB games but has been fixed with https://github.com/tonywagner/mlbserver/commit/eb5ad1f303944eefb2105e91fb1d6b72ec276bae

**Comment by cr08** [comment]
I have been up to date. The version I was on showed as 6.29 before I updated to the new revision you just made. Tried some other teams that are still playing this evening and same thing. But I totally forgot about and rediscovered the Scan Mode option and once I turned that off everything seems to be fine now.

COL will be having a double header tomorrow so will check and see if it all works. But so far looks like everything's good. Simple fix. I'll report back if there are any further issues.

Thanks again! 🙂
